src/main/java/com/kodilla/ecommercee/controller/CartController.java

```diff
@@ -2,52 +2,66 @@
 
 import com.kodilla.ecommercee.domain.dto.CartDto;
 import com.kodilla.ecommercee.domain.dto.CartItemDto;
-import lombok.RequiredArgsConstructor;
+import com.kodilla.ecommercee.domain.dto.OrderDto;
+import org.springframework.http.HttpStatus;
 import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
-
 import java.math.BigDecimal;
+import java.time.LocalDate;
 import java.util.ArrayList;
 import java.util.List;
 
+
 @RestController
 @CrossOrigin("*")
-@RequestMapping(consumes = MediaType.APPLICATION_JSON_VALUE, value = "/carts")
+@RequestMapping("/carts")
 public class CartController {
     
-    @PostMapping(value = "/createCart/{userId}")
-    public ResponseEntity<Void> createCart(@PathVariable Long userId) {
-        List<CartItemDto> cart = new ArrayList<>();
-        //post to repository//
-        return ResponseEntity.ok().build();
+    @PostMapping(consumes = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<String> createCart(@RequestBody CartDto cartDto) {
+        return new ResponseEntity<>("New cart was created for userid: " + cartDto.getUserId(), HttpStatus.OK);
     }
     
-    @GetMapping(value = "/{userId}/{cartId}")
-    public ResponseEntity<CartDto> getCart(@PathVariable Long userId, @PathVariable Long cartId) {
-        //get from repository//
-        //map to Dto//
-        CartDto currentCard = new CartDto(userId, cartId, new ArrayList<>());
-        return ResponseEntity.ok(currentCard);
+    @GetMapping("/products")
+    public ResponseEntity<List<CartItemDto>> getCart(@RequestParam long cartId) {
+        List<CartItemDto> cartItemDtos = new ArrayList<>();
+        cartItemDtos.add(new CartItemDto(
+                1L,"product1","description1", new BigDecimal(100),4L));
+        cartItemDtos.add(new CartItemDto(
+                2L,"product2","description2", new BigDecimal(200),3L));
+        return ResponseEntity.ok(cartItemDtos);
     }
 
-    @PostMapping(value = "{cartId}")
-    public ResponseEntity<Void> addCartItem(@PathVariable Long cartId, @RequestBody CartItemDto cartItemDto) {
-        //post item into Cart//
-        return ResponseEntity.ok().build();
+    @PutMapping("/add/{cartId}")
```

Comment:
I think that if we don't have to use cartId in method removeProductToCart then here also but for now we must decide if it's ok

Reply:
I'm not sure ether but I think when we will be doing method in java we will need that parameter